proxysql/dockerdir/usr/bin/add_pxc_nodes.sh

```diff
@@ -54,6 +54,8 @@ function main() {
         SSL_ARG="--use-ssl=yes"
     fi
 
+    sed "s/^WRITE_NODE=.*/WRITE_NODE='$first_host'/g" /etc/proxysql-admin.cnf 1<> /etc/proxysql-admin.cnf
```

Comment:
It doesn't work, regex is searching from start of line because of `^` and WRITE_NODE is not in the start of line. `^WRITE_NODE=.*` should be `WRITE_NODE=.*`

Reply:
fixed